source/api/events/command-start-event.md

```diff
@@ -0,0 +1,53 @@
+---
+title: command:start Event
+---
+
+The `command:start` event fires when cy begins actually running and executing your command. Useful for debugging and understanding how the command queue is async.
+
+# Environment
+
+Occurs in the **browser** and in the {% url "background process" background-process %}.
+
+# Arguments
+
+* command **(Object)**
+
+# Usage
+
+## In the browser
+
+In the browser, the argument is the actual command instance.
+
+```javascript
+// in a test or cypress/support/index.js
+
+Cypress.on('command:start', (command) => {
+  command.get('name') // => 'get'
+  command.get('args') // => ['#foo']
+  command.get('subject') // => jQuery(<div#foo />)
+  // command instance has the following properties you can 'get':
+  // - name
+  // - args
+  // - subject
+  // - type
+  // ... and more ...
+})
```

Comment:
I'll try to come up with a good example, but I'm honestly not sure I can 😅 Some of these events have very clear use cases, but others like this we're sort of throwing out there like "Hey, this might be useful. It's here if you find a use for it."

Reply:
Totes Chris, I feel ya.